### PR TITLE
Fix bug in showGradeAfterSubmission,

### PR DESCRIPTION
**Pull request by TaliHa**
Action clone of BaseEntry is now with a new parameter cloneOptions that defines the properties that should be copied in the clone operation. Currently only 'Categories' clone option is supported
PLAT-4058 #2 days

### CODE REVIEW

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 1]
```diff
@@ -1097,7 +1097,17 @@ public static function copyEntry(entry $entry, Partner $toPartner = null, $dontC
 				$shouldCopyDataForClip = true;
 			   }
 		}
-		
+
+		$bCopy = true;
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Give more meaningful name to the flag bCopy, shouldCopyCategory

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 2]
```diff
@@ -1120,44 +1130,40 @@ public static function copyEntry(entry $entry, Partner $toPartner = null, $dontC
...
-			                }
-			                $entryFilter->fields["_in_category_ancestor_id"] = implode (",", $newCategoriesIds);
-			            }
-			            $toEntryFilterXML = $toFiltersXml->addChild("filter");
-			            $toEntryFilterXML = $entryFilter->toXml($toEntryFilterXML);
-		            }
-		            
+					if ($bCopy)
+					{
+						foreach ($fromFiltersList as $filterXML) {
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Indentation to space

File: alpha/apps/kaltura/lib/myEntryUtils.class.php [hunk 3]
```diff
@@ -1097,7 +1097,17 @@ public static function copyEntry(entry $entry, Partner $toPartner = null, $dontC
 				$shouldCopyDataForClip = true;
 			   }
 		}
-		
+
+		$bCopy = true;
+		if (is_null($cloneOptions))
```
Comment (MosheMaorKaltura) [inline alpha/apps/kaltura/lib/myEntryUtils.class.php]: Simplify the condition check

**Comment by MosheMaorKaltura** [comment]
Tali,
All the changes to file myEntryUtils.php are missing.

**Comment by tan-tan-kanarek** [comment]
In the future please use different pull-requests for different changes.

**Pull request closed without merging**